File: app/src/main/java/com/androidevlinux/percy/UTXO/data/models/bitstamp/BitstampBean.java
package com.androidevlinux.percy.UTXO.data.models.bitstamp;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class BitstampBean implements Serializable {

    @SerializedName("high")
    @Expose
    private String high;
    @SerializedName("last")
    @Expose
    private String last;
    @SerializedName("timestamp")
    @Expose
    private String timestamp;
    @SerializedName("bid")
    @Expose
    private String bid;
    @SerializedName("vwap")
    @Expose
    private String vwap;
    @SerializedName("volume")
    @Expose
    private String volume;
    @SerializedName("low")
    @Expose
    private String low;
    @SerializedName("ask")
    @Expose
    private String ask;
    @SerializedName("open")
    @Expose
    private String open;
    private final static long serialVersionUID = -5266039118653418081L;

    public String getHigh() {
        return high;
    }

    public void setHigh(String high) {
        this.high = high;
    }

    public BitstampBean withHigh(String high) {
        this.high = high;
        return this;
    }

    public String getLast() {
        return last;
    }

    public void setLast(String last) {
        this.last = last;
    }

    public BitstampBean withLast(String last) {
        this.last = last;
        return this;
    }

    public String getTimestamp() {
        return timestamp;
    }

    public void setTimestamp(String timestamp) {
        this.timestamp = timestamp;
    }

    public BitstampBean withTimestamp(String timestamp) {
        this.timestamp = timestamp;
        return this;
    }

    public String getBid() {
        return bid;
    }

    public void setBid(String bid) {
        this.bid = bid;
    }

    public BitstampBean withBid(String bid) {
        this.bid = bid;
        return this;
    }

    public String getVwap() {
        return vwap;
    }

    public void setVwap(String vwap) {
        this.vwap = vwap;
    }

    public BitstampBean withVwap(String vwap) {
        this.vwap = vwap;
        return this;
    }

    public String getVolume() {
        return volume;
    }

    public void setVolume(String volume) {
        this.volume = volume;
    }

    public BitstampBean withVolume(String volume) {
        this.volume = volume;
        return this;
    }

    public String getLow() {
        return low;
    }

    public void setLow(String low) {
        this.low = low;
    }

    public BitstampBean withLow(String low) {
        this.low = low;
        return this;
    }

    public String getAsk() {
        return ask;
    }

    public void setAsk(String ask) {
        this.ask = ask;
    }

    public BitstampBean withAsk(String ask) {
        this.ask = ask;
        return this;
    }

    public String getOpen() {
        return open;
    }

    public void setOpen(String open) {
        this.open = open;
    }

    public BitstampBean withOpen(String open) {
        this.open = open;
        return this;
    }

}

File: app/src/main/java/com/androidevlinux/percy/UTXO/data/models/bitstamp/BitstampBean.kt
package com.androidevlinux.percy.UTXO.data.models.bitstamp

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import java.io.Serializable

class BitstampBean : Serializable {

    @SerializedName("high")
    @Expose
    var high: String? = null
    @SerializedName("last")
    @Expose
    var last: String? = null
    @SerializedName("timestamp")
    @Expose
    var timestamp: String? = null
    @SerializedName("bid")
    @Expose
    var bid: String? = null
    @SerializedName("vwap")
    @Expose
    var vwap: String? = null
    @SerializedName("volume")
    @Expose
    var volume: String? = null
    @SerializedName("low")
    @Expose
    var low: String? = null
    @SerializedName("ask")
    @Expose
    var ask: String? = null
    @SerializedName("open")
    @Expose
    var open: String? = null

    fun withHigh(high: String): BitstampBean {
        this.high = high
        return this
    }

    fun withLast(last: String): BitstampBean {
        this.last = last
        return this
    }

    fun withTimestamp(timestamp: String): BitstampBean {
        this.timestamp = timestamp
        return this
    }

    fun withBid(bid: String): BitstampBean {
        this.bid = bid
        return this
    }

    fun withVwap(vwap: String): BitstampBean {
        this.vwap = vwap
        return this
    }

    fun withVolume(volume: String): BitstampBean {
        this.volume = volume
        return this
    }

    fun withLow(low: String): BitstampBean {
        this.low = low
        return this
    }

    fun withAsk(ask: String): BitstampBean {
        this.ask = ask
        return this
    }

    fun withOpen(open: String): BitstampBean {
        this.open = open
        return this
    }

    companion object {
        private const val serialVersionUID = -5266039118653418081L
    }

}
